src/Components/SearchBox.tsx

```diff
@@ -2,7 +2,7 @@
 import React from "react";
 import TextField from "@material-ui/core/TextField";
 import Autocomplete from "@material-ui/lab/Autocomplete";
-import { buildSearchBox, SearchBoxState } from "@coveo/headless";
+import { buildSearchBox, SearchBoxState, searchBoxOptionsSchema } from "@coveo/headless";
```

Comment:
Hmm, we should not allow importing the schema from headless. It is supposed to be internal. We have a PR fixing this in a later version, but I am curious why you are importing it here?

Reply:
It's something we forgot to delete when were debugging - we'll fix it in the next PR we make.